client/metrics.go

```diff
@@ -0,0 +1,14 @@
+package client
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	metricClients = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "hydra",
```

Comment:
Would it make sense to have the namespace fixed as a const somewhere? We should also decide if it's `ory-hydra`, `ory_hydra`, `ORY Hydra`, `hydra` or any other combination.